examples/examples_dotnet_test.go

```diff
@@ -42,6 +42,7 @@ func TestDotNet(t *testing.T) {
 }
 
 func TestAzureContainerRegistryDotNet(t *testing.T) {
+	t.Skipf("skip to rule out dotnet")
```

Comment:
Are these skips intentional? If so, why?

Reply:
I ran into unexpected behavior for one of the docker cli methods. This was part of the debugging process. Should be unskipped now! 👍